### ISSUE TITLE
Manual backup is not starting - Alpha/Beta-Software ahead! 

### ISSUE THREAD

**Issue by Grothesk242** [simatec/ioBroker.backitup#915]
**Describe the bug**  
Clicking on 'iobroker Backup starten' opens the usual pop up: 
Started iobroker ...

But nothing does happen and in the LogFile it shows: 

```
2023-03-30 19:25:40.655  - error: backitup.0 (408580) Error: Parameter "options" needs to be a real object but type "string" has been passed
```

**Versions:**  
 - Adapter version: 2.6.15
 - JS-Controller version: 5.0.1-alpha.0-20230330-e0b409fe
 - Node version: v19.8.1
 - Operating system: Debian 12 'Bookworm'

**Comment by simatec** [comment]
Fix for js-controller 5 is on Github... Release is in work

**Comment by Grothesk242** [comment]
```
iobroker.backitup@2.6.15 (git+ssh://git@github.com/simatec/ioBroker.backitup.git#8fbf70f2985ce81516d49d6b7de5287a7b4ce9ae)
```
is working! 
SMB-Version had been reset to v1.0 though. Wasn't 3.1.1 the default?

**Comment by simatec** [comment]
Default SMB is 3.1.1
https://github.com/simatec/ioBroker.backitup/blob/8fbf70f2985ce81516d49d6b7de5287a7b4ce9ae/io-package.json#L342

**Comment by Grothesk242** [comment]
Hmm. I dunno who did set it back to 1.0 then... :-)

**Comment by simatec** [comment]
Fix in v2.6.16
